pkg/cmd/server/api/types.go

```diff
@@ -6,6 +6,14 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 )
 
+type OpenShiftFeature int
```

Comment:
I definitely prefer strings.  It makes the config file read cleaner if they're ever used and it avoid weird reordering problems.  Otherwise, the position in this list must forever remain the same across all branches, etc.